pages/terms/index.html

```diff
@@ -0,0 +1,66 @@
+---
+layout: default
+title: Terms of Service
+modified_date: July 18, 2018
+permalink: /terms/
+---
+
+<div class="main">
+  <div class="section section-hero">
+    <div class="container">
+      <div class="row">
+        <div class="col-xs-12">
+          <h1>{{ page.title }}</h1>
+          <h2 class="h3">This page outlines the terms of service and use of Gruntwork’s products and services.</h2>
+          <p><strong>It’s important to understand these Terms of Service, as they define your rights and our rights in our relationship.</strong> But legalese is not always easy to understand, so we’ve provided “plain English” summaries on the left side of the page to help you understand the official legal language on the right side of the page. Remember that we’re providing you these summaries for informational purposes only. Our actual Terms are the ones on the right under "Legalese" and will apply if we ever need to work through any issues.</p>
+          <p><em>Last Modified: {{ page.modified_date }}</em></p>
+        </div>
+      </div>
+      <div class="row ">
+        <div class="col-xs-12 col-sm-offset-3 col-sm-9 visible-xs-block visible-sm-block">
+          <div class="switch-group">
+            <label class="switch">
+              <input type="checkbox" id="human" data-switch="" name="humanese">
+              <span class="slider round"></span>
+            </label>
+            <span class="switch-label small bold" title="">Translate legalese to human-friendly</span>
+          </div>
+        </div>
+        <div class="col-xs-12 visible-md-block visible-lg-block">
+            <div class="row terms-head">
+                <h5 class="h2 col-md-5">Plain English</h5>
+                <h5 class="h2 col-md-7">Legalese</h5>
+            </div>
+        </div>
+        <div class="col-xs-12 tos-table">
+          {% for category in site.data.terms-of-service %}
```

Comment:
Thx for putting the terms in a data file. Would be painful to edit a gigantic HTML file.